plugins/wacom-usb/fu-wac-module-scaler.c

```diff
@@ -92,7 +92,10 @@ fu_wac_module_scaler_write_firmware(FuDevice *device,
 
 	/* build each data packet */
 	data = g_bytes_get_data(fw, &len);
+	fu_memwrite_uint32(buf_start, len, G_LITTLE_ENDIAN);
```

Comment:
What happens if you send the length to an old firmware that's expecting `0x00000000` i.e. `FU_WAC_MODULE_SCALER_START_NORMAL`?

Reply:
The old products and their firmware do not care the field, so whatever is written in "buf_start", they don't check the first four bytes.

Reply:
Ohh, in that case can we remove:

    #define FU_WAC_MODULE_SCALER_START_NORMAL    0x00
    #define FU_WAC_MODULE_SCALER_START_FULLERASE 0xFE

...also please.

Reply:
Thanks for your catch, I removed those define macros.